apps/web/src/pages/User/index.tsx

```diff
@@ -27,8 +29,9 @@ const User = () => {
   });
 
   const { user } = data;
-
+  /** added a button on line 46 and added the followers text on lines 49-51 */
```

Comment:
I think it's great that you can want to make things clear, but you can remove this comment.